certified-connectors/Business Assist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,688 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Business Assist",
+    "description": "Business Assist APIs are using Machine Learning models which will help in improving the customer experience and provide customer success. These APIs will help you optimize your customer support that can help you forecast call volumes, analyze customer feedback, and provide self-help support for faster service. You can integrate with any platform they already use, allowing them to create their own technology stack based on both preference and value.",
+    "version": "1.0",
+    "contact": {
+      "name": "Microsoft",
+      "url": "https://admin.microsoft.com",
+      "email": "PowerConnector-BA@microsoft.com"
+    }
+  },
+  "host": "businessassist.microsoft.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/forecast": {},
+    "/api/beta/forecast/{Forecast job Id}": {
+      "get": {
+        "responses": {
+          "202": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "OperationId": {
+                  "type": "string",
+                  "description": "Forecast operation Id"
+                },
+                "Name": {
+                  "type": "string",
+                  "description": "Forecast name"
+                },
+                "Status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Forecast status"
+                },
+                "Diagnostics": {
+                  "type": "string",
+                  "description": "Diagnostics message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get forecast status",
+        "description": "Get forecast job status by job id.",
+        "operationId": "GetForecastStatus",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Forecast job Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "a6fe7af6-ba42-4676-9a4c-901d0bd44d19",
```

Comment:
@sjagaleti please change the value of the description. Secret look alike values are not allowed.